retrofit-adapters/guava/src/main/java/retrofit2/adapter/guava/GuavaCallAdapterFactory.java

```diff
@@ -75,8 +78,8 @@ private GuavaCallAdapterFactory() {
 
     // Generic type is Response<T>. Extract T and create the Response version of the adapter.
     if (!(innerType instanceof ParameterizedType)) {
-      throw new IllegalStateException("Response must be parameterized"
-          + " as Response<Foo> or Response<? extends Foo>");
+      throw new IllegalStateException(
+          "Response must be parameterized" + " as Response<Foo> or Response<? extends Foo>");
```

Comment:
supernit: the + is now weird

Reply:
yeah I noticed this as well. I'll check and file bugs for some of these.